curriculum/challenges/english/08-coding-interview-prep/rosetta-code/longest-string-challenge.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 5e6dd14192286d95fc43046e
+title: Longest string challenge
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+In the longest string challenge, you have to find all the strings that have the longest lengths among all the given strings.
```

Comment:
I think this could be rephrased for clarity

Reply:
@thecodingaviator Do you think this would be better:
In this challenge, you have to find the strings that are the longest among the given strings.